src/ContainerTerminal.jsx

```diff
@@ -227,21 +228,20 @@ class ContainerTerminal extends React.Component {
         this.disconnectChannel();
         if (this.state.channel)
             this.state.channel.close();
-        this.state.term.dispose();
+        this.term.dispose();
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1358-20230718-052024-6af1ca09-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)